openapi/paths/risk-scores.yaml

```
@@ -0,0 +1,62 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Risk Score
+  summary: Retrieve risk score rules
+  operationId: GetRiskScoreRules
+  x-sdk-operation-name: getAll
+  security:
+    - SecretApiKey: []
+    - JWT: []
+    - ApplicationJWT: []
+  description: Retrieve risk score rule.
+  responses:
+    '200':
+      description: Risk score rules retrieved.
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/RiskScore/RiskScoreRules.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+put:
+  x-products:
+    - Core
+  tags:
+    - Risk Score
+  summary: Change Risk Score rules
+  operationId: PutRiskScoreRules
+  x-sdk-operation-name: updateRiskScoreRules
+  description: Modify Risk Score rules.
```

issue: [based on the style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations)

> Start all operation descriptions with an active verb (Retrieves, Adds, Creates, Updates, Deletes, Sets, Specifies ...)

```suggestion
  description: Modifies risk score rules.
```